test/functorch/test_aotdispatch.py

```diff
@@ -731,6 +731,23 @@ def f(a):
         # by the time it becomes a graph input. Good to test both cases.
         self.verify_aot_autograd(f, inp, test_mutation=True)
 
+    def test_input_mutation_hidden_from_autograd_aliasing(self):
+        def f(a):
+            a_alias = a.view(-1)
+            with torch.no_grad():
+                a_alias.mul_(2)
+            return a + 1
+        inp = [torch.ones(4, requires_grad=True)]
+        fw_graph = self.verify_aot_autograd(f, inp, test_mutation=True, only_keep_inference_mutations=True)
+        self.assertExpectedInline(fw_graph.code.strip(), """\
```

Comment:
In tests like this it's helpful to mention what exactly we're looking for in the graph. In this case, I believe it's that there should be a copy_ in the graph

Reply:
Definitely agreed, I'll call it out (the important bit is indeed that we get a copy_() inside the graph, and not in the opaque epilogue)